### PR TITLE
Addition of Board Order V. Risk Management Duties

### PR DESCRIPTION
**Pull request by akhilvelpuri**

### CODE REVIEW

File: board_orders.md [hunk 1]
```diff
@@ -126,3 +126,16 @@ In the event the Fraternity makes a donation or other contribution to a non-prof
...
+
+The following is a list of things that all RMs should look out for/do while on duty:
+* Illegal substances of any kind.
+* Fire in the fire pit, stove or anywhere else.
+* Destruction/damage of house property.
+* Non-LAN members accessing the fridge, or food on the shelves in the kitchen.
+* Non-LAN members accessing the second floor, or back house.
+* Over-intoxicated or aggressive behavior. 
+* Appropriately marking the underage people attending the social events.
+* In the event that the cops show up, RM’s should alert each other (and possibly other officers nearby) and be the point of contact with them.
```
Comment (aaaanderson) [inline board_orders.md]: Breaking out this line to a separate paragraph could be clearer. We could take this opportunity to say exactly who is the main point of contact for the authorities (and who should point authorities towards someone else).

File: board_orders.md [hunk 2]
```diff
@@ -126,3 +126,16 @@ In the event the Fraternity makes a donation or other contribution to a non-prof
     * During that meeting, all elected positions will be discussed in order, as described by the Bylaws.
     * Each candidate of the current position will have 5 minutes to give a speech.
     * After each speech is given, all candidates of the position must leave the room and the rest of the membership will have 5 minutes to talk about the candidates.
+    
+## Board Order V. Risk Management Duties
+
+The following is a list of things that all RMs should look out for/do while on duty:
```
Comment (aaaanderson) [inline board_orders.md]: We should clarify the definition of who we're talking about before the list.

For example:

> One or more Risk Management officers or their previously announced designee should be present and completely sober at all official LAN events. For the purposes of this Order, an "Acting Risk Manager" is this officer or their previously-announced designee.

**Comment by ThomasGaubert** [comment]
In addition to Andy's comments I think it'd be a good idea to include some sort of guidelines as to number of RMs based on the number of people at an event.

**Comment by ThomasGaubert** [comment]
@akhilvelpuri Once you fix the wording on [line 137](https://github.com/TexasLAN/documents/pull/17/files#diff-9f3eb1975fac378efb77a9fe418ac8c8R137) (no one should access fridge, etc.) we can merge this in.

**Pull request merged**